templates/overview.html

```diff
@@ -20,10 +20,10 @@
 
     {%- include head-polyfills.html -%}
   </head>
-  <body class="hxApp">
-    <div class="hxApp__container overview-wrapper">
+  <body class="hxVertical">
+    <div class="app overview-wrapper">
       {%- include nav.html -%}
-      <main class="hxApp__content overview-content">
+      <main class="stage overview-content">
```

Comment:
* `id="stage"` not `class`
* incorrect element/id nesting for layout (`<main id="content">` is nested within `<div id="stage">`)

```html
<body class="hxVertical">
  <div id="app">
    <div id="stage">
      <main id="content">...</main>
    </div>
  </div>
</body>
```